resources/views/content/top-links.blade.php

```diff
@@ -0,0 +1,41 @@
+<x-section class="mb-3 flex flex-col gap-2">
+    <a class="btn text-center py-2" href="/globalRanking.php?s=5&t=2">
+        <x-fas-medal/>
+        Global Ranking
+    </a>
+    <a class="btn text-center py-2" href="{{ route('download.index') }}">
+        <x-fas-download/>
+        Download Emulator
+    </a>
+    @if(config('services.discord.invite_id'))
+        <a class="btn text-center py-2" href="https://discord.gg/{{ config('services.discord.invite_id') }}">
+            <x-fab-discord/>
+            Join us on Discord
+        </a>
+    @endif
+    @if(config('services.patreon.user_id'))
+        <a class="btn text-center py-2" href="https://www.patreon.com/bePatron?u={{ config('services.patreon.user_id') }}">
+            <x-fab-patreon/>
+            Become a Patron
+        </a>
+    @endif
+    <div class="flex gap-1">
+        <a class="btn text-center py-2 grow" href="https://news.retroachievements.org/">
+            <x-fas-newspaper/>
+            RANews
+        </a>
+        <a class="btn text-center py-2 grow" href="https://www.youtube.com/channel/UCIGdJGxrzmNYMaAGPsk2sIA">
+            <x-fas-microphone/>
+            RAPodcast
+        </a>
+    </div>
+    <div class="flex gap-1">
+        <a class="btn text-center py-2 grow" href='https://docs.retroachievements.org/'>
+            <x-fas-book/>
+            Documentation
+        </a>
+        <a class="btn text-center py-2 grow" href='https://docs.retroachievements.org/FAQ/'>
+            FAQ
+        </a>
+    </div>
+</x-section>
```

Comment:
I kind of miss the colored emoticons.
![image](https://user-images.githubusercontent.com/32680403/215285595-44723612-420c-4567-ad05-c4b306341529.png)

Reply:
I was pretty sure that this would be a controversial change :)
I think it looks better with emojis, too. 

However, I wished emojis weren't introduced in the first place as they are wildly inconsistent visually across systems.

![Screenshot 2023-01-28 at 22 30 22](https://user-images.githubusercontent.com/1280590/215292077-95e63fce-4af6-475d-8947-6c0b32a48d3c.png)

The reason for moving back to consistent iconography for me is to establish visual consistency again before introducing emojis anywhere. The proper handling in web would be to provide a defined set of emojis regardless of operating system and/or emoji support.

Not sure how to move forward with this ... I could add some color to the icons?

Reply:
With colored/branded icons:

![Screenshot 2023-01-28 at 22 49 34](https://user-images.githubusercontent.com/1280590/215292654-9b433b61-8866-41ee-9b03-701f2fb799eb.png)

Reply:
I think a grid-like view would be even nicer:

![image-3](https://user-images.githubusercontent.com/22218549/215335631-925c3160-5207-457a-b335-dbebdca8a797.png)
![image-4](https://user-images.githubusercontent.com/22218549/215335651-05523ebb-5dc7-4781-a88c-3f243314e742.png)

Reply:
The layout of the links can be changed in a separate PR (not sure about the grid yet).
This is mainly about replacing emojis with icons.

One of the problems of using emojis, apart from kerning inconsistencies across systems:
![Screenshot 2023-01-29 at 16 30 52](https://user-images.githubusercontent.com/1280590/215336891-74f180c7-6c72-4440-8394-b9a160b9b9ee.png)
<img width="324" alt="Screenshot 2023-01-29 at 16 31 17" src="https://user-images.githubusercontent.com/1280590/215336895-371c9ea6-2709-49dc-9b29-d3cab04664f3.png">

Reply:
I'm totally for replacing emoji with icons, and I think adding color is enough for them to look nice(r). Plus, even if we wanted non-monochrome emoji SVG icons, we could just pick them from a icon set ([like this one](https://icons8.com/icon/set/emoji/emoji)), right?

So it's more a technical thing than about visuals (with the bonus of SVG presentation being the same for all users).

Reply:
Using on of the available emoji sets would be a good solution, yes; rendering any occurrence of an emoji using said set.
As for icons: we have all of those available in blade: https://blade-ui-kit.com/blade-icons

Reply:
Resolving this discussion; as discussed, adding a different layout to the top links may follow in another PR.